README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
When I move industrial_robot_simulator from service side to client mode. I encountered a weird issue that when I operator on the client side, like scan or blend, the robot(simulated) needs to wait around 40 seconds then start to move.

This is what I changed:
Before, server side will launch /opt/ros/indigo/share/industrial_robot_simulator/launch/robot_interface_simulator.launch

>   <!-- industrial_robot_simulator: accepts robot commands and reports status -->
>   <node pkg="industrial_robot_simulator" type="industrial_robot_simulator" name="industrial_robot_simulator"/>
> 
>   <!-- joint_trajectory_action: provides actionlib interface for high-level robot control -->
>   <node pkg="industrial_robot_client" type="joint_trajectory_action" name="joint_trajectory_action"/>
> 
> 

Now 
server side will only run:

> >  <!-- joint_trajectory_action: provides actionlib interface for high-level robot control -->
> >   <node pkg="industrial_robot_client" type="joint_trajectory_action" name="joint_trajectory_action"/>
> 

Client side will run :
 
> <!-- industrial_robot_simulator: accepts robot commands and reports status -->
>   <node pkg="industrial_robot_simulator" type="industrial_robot_simulator" name="industrial_robot_simulator"/>
> 

I don't think the delay comes from network, since client machine and server machine works at same lab and connects to same switch.
And also after I move industrial_robot_simulator back to service side, the problem will disappear.

I'm working on this now, will let you know if I get any progress?